librocksdb_sys/crocksdb/c.cc

```diff
@@ -6383,10 +6412,38 @@ void crocksdb_run_sst_dump_tool(int argc, char** argv,
 }
 
 /* Titan */
+struct ctitandb_checkpoint_t {
```

Comment:
Only one type is necessary. Titan checkpoint is just a subclass of checkpoint.

Reply:
https://github.com/tikv/titan/blob/master/include/titan/checkpoint.h#L8-L35
it's not implemented as a subclass. or am I missing something?

Reply:
Got it, I misread the code.